File: server/Models/Ingredient.cs
namespace wk10.Models;

public class Ingredient
{
  public int Id { get; set; }
  public string Name { get; set; }
  public string Quantity { get; set; }
  public string CreatorId { get; set; }
  public uint RecipeId { get; set; }
  public Recipe Recipe { get; set; }
}

File: client/src/models/Ingredient.js
export class Ingredient {
  constructor(data) {
    this.id = data.id
    this.name = data.name
    this.quantity = data.quantity
    this.creatorId = data.creatorId
    this.recipeId = data.recipeId
    this.recipe = data.recipe
    this.edit = false
  }
}